Automattic/wp-calypso
# SMP: Replace arrow down with upload icon for importing a site

Related to https://github.com/Automattic/wp-calypso/issues/84870

## Proposed Changes

This PR replaces the arrow down icon with the Gutenberg download icon for `Import an existing site` action on `/sites` page.

<img width="1180" alt="Screenshot 2023-12-21 at 11 13 31 AM" src="https://github.com/Automattic/wp-calypso/assets/25575134/85e995fb-bdaf-46ca-85b9-beb96c54148d">

## Testing Instructions

* Navigate to Calypso Live Link
* Navigate to `/sites` page
* Click on the chevron by the `Add new site` button in the top right corner 
* Observe that dropdown with the `Impost an existing site` action
* Observe that the icon has been updated to `download` 
* Confirm that the icon is properly aligned

## Pre-merge Checklist
